docs_app/src/app/custom-elements/migration-timeline/data-access/migration-timeline.ts

```diff
@@ -0,0 +1,1517 @@
+        breakingVersion: '8',
+        breakingLink: 'breakingChange-class-Observable-create-removed',
+        headline: 'Observable static method `create` deprecated in favour of normal instantiation over `new Observable()`',
+        reason: `After moving to "pipeable" operators \`create\` static Observable method got deprecated.
+        No new static method was created because \`new Observable() is more intuitive and natural to the language.
+        Technically older versions of TypeScript had many more limitations that today's version`,
+        implication: '@!TODO',
+        exampleBefore: `
+        import { Observable } from 'rxjs';
+        Observable.create(subscriberFunc)
+          .subscribe({(n) => console.log(n)});
+        `,
+        exampleAfter: `
+        import { Observable } from 'rxjs';
+        new Observable(subscriberFunc)
+          .subscribe({(n) => console.log(n)});
+        `
+      },
+      {
+        subject: 'TimeInterval',
+        subjectApiSymbol: ApiSymbols.class,
+        linkName: 'deprecation-class-TimeInterval-to-interface',
+        type: 'deprecation',
+        sourceLink: 'https://github.com/ReactiveX/rxjs/blob/6.4.0/src/internal/operators/timeInterval.ts#L69',
+        breakingVersion: '8',
+        breakingLink: 'breakingChange-class-TimeInterval-removed',
+        headline: 'Class `TimeInterval` deprecated. As it is an internal implementation detail use it as Interface only.',
+        reason: `Class TimeInterval gets deprecated in favour of
+         interface [TimeInterval](https://github.com/ReactiveX/rxjs/blob/6.5.3/src/internal/types.ts#L19-L22)`,
+        implication: '@!TODO',
+        exampleBefore: `@TODO`,
+        exampleAfter: `@TODO`
+      }
+    ],
+    breakingChanges: []
+  },
+  {
+    version: '6.5.0',
+    date: '2019-04-23',
+    deprecations: [
+      {
+        subject: 'of',
+        subjectApiSymbol: ApiSymbols.function,
+        linkName: 'deprecation-static-of-scheduler',
+        type: 'deprecation',
+        sourceLink: 'https://github.com/ReactiveX/rxjs/blob/6.5.0/src/internal/observable/of.ts#L29',
+        breakingVersion: '8',
+        breakingLink: 'breakingChange-static-of-scheduler-removed',
+        headline: 'Static `of` deprecated the scheduler argument.',
+        reason: 'The scheduling API is heavy and rarely used. Therefor it will get released as a separate package.' +
+          'If you used `of` with a scheduler argument, you can use {@link scheduled} instead.',
+        implication: '@!TODO',
+        exampleBefore: `
+        import { of, asyncScheduler } from 'rxjs';
+        of(777, asyncScheduler)
+          .subscribe({next: (n) => { console.log(a); } });
+        `,
+        exampleAfter: `
+        import { scheduled, asyncScheduler } from 'rxjs';
+        scheduled([777], asyncScheduler)
+          .subscribe({next: (n) => { console.log(a); } });
+        `
+      },
+      {
+        subject: 'combineLatest',
```

Comment:
Regarding the 'implication', I think it should be an explanation that accompanies the 'before deprecation' and 'after deprecation' snippets. That is, it should explain what's different between the two so that users don't have to try to spot the differences themselves.

## Reason

The reason doesn't seem quite right, to me.

My understanding is not that scheduling is going to be released as a separate package, rather that it is only going to be used where appropriate.

ATM, there are signatures all over the place that include optional scheduler parameters.

The means that the implementation needs to check for a scheduler, etc., which references/imports the scheduling and prevents it from being tree-shaken.

## Implication

I would say something like:

> For `combineLatest`, the removal of the `scheduler` parameter means that if callers want notifications to be scheduled, they will have to use `observeOn`.

## Before Derecation

```ts
import { combineLatest, asyncScheduler } from "rxjs";
combineLatest(a$, b$, asyncScheduler).subscribe({
  next: ([a, b]) => console.log(a, b)
});
```

## After Deprecation

```ts
import { combineLatest, asyncScheduler, observeOn } from "rxjs";
combineLatest(a$, b$).pipe(
  observeOn(asyncScheduler)
).subscribe({
  next: ([a, b]) => console.log(a, b)
});
```